src/languages/php.ts

```diff
@@ -1,14 +1,13 @@
-import { insertBefore } from '../shared/language-util';
 import { embeddedIn } from '../shared/languages/templating';
 import { tokenize } from '../shared/symbols';
 import markup from './markup';
-import type { LanguageProto } from '../types';
+import type { Grammar, LanguageProto } from '../types';
 
 export default {
 	id: 'php',
-	require: markup,
+	base: markup,
```

Comment:
So far I've been using *both* `require` and `base`, we can later remove `require` but some languages do have requires beyond their base.

Reply:
```suggestion
	require: markup,
	base: markup,
```